src/components/Editor/index.js

```diff
@@ -415,48 +379,17 @@ class Editor extends PureComponent<Props, State> {
     return this.props.closeConditionalPanel();
   };
 
-  // If the location has changed and a specific line is requested,
-  // move to that line and flash it.
-  highlightLine() {
-    const { selectedLocation, selectedFrame } = this.props;
-    if (!selectedLocation) {
-      return;
-    }
-
-    // Make sure to clean up after ourselves. Not only does this
-    // cancel any existing animation, but it avoids it from
-    // happening ever again (in case CodeMirror re-applies the
-    // class, etc).
-    if (this.lastJumpLine !== null) {
-      clearLineClass(this.state.editor.codeMirror, "highlight-line");
-    }
-
-    let line = null;
-    if (selectedLocation.line >= 0) {
-      line = this.scrollToPosition();
-    }
+  scrollToLocation(nextProps) {
```

Comment:
this function is causing us to jump a bit... so not quite ready for showtime

Reply:
cleaned it up by using `createLocation` in selectLocation so everything had a nice default.